<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>testClass</class>
 <widget class="QMainWindow" name="testClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>623</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>test</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow { background-color: #F5F5DC; }
QWidget { color: #000000; }
QLineEdit{
    border: 1px solid #000000;
    color: #000000;
    background: #FFFFFF;
    border-radius: 10px;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widgetChart1" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>351</width>
      <height>271</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
    border: 2px inset #888888;
    color: #000000;
    background: #FFFFFF;
    border-radius: 10px;
    padding: 2px 8px;
    font-weight: bold;
	border-radius: 8px;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zoom_reset">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>310</y>
      <width>108</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #fdfdfd, stop:1 #c3c3c3
    );
    min-width: 80px;
    min-height: 28px;
    color: #222222;
    font-weight: bold;
    padding: 4px 12px;
    /* 轻微外阴影，增强立体感 */
    box-shadow: 1px 1px 3px #b0b0b0;
}
QPushButton:hover {
    background: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #eaeaea, stop:1 #bdbdbd
    );
    border: 2px solid #5a5a5a;
}
QPushButton:pressed {
    background: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #c3c3c3, stop:1 #fdfdfd
    );
    border: 3px inset #5a5a5a;
}</string>
    </property>
    <property name="text">
     <string>复位</string>
    </property>
   </widget>
   <widget class="QPushButton" name="display_xy">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>310</y>
      <width>108</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #fdfdfd, stop:1 #c3c3c3
    );
    min-width: 80px;
    min-height: 28px;
    color: #222222;
    font-weight: bold;
    padding: 4px 12px;
    /* 轻微外阴影，增强立体感 */
    box-shadow: 1px 1px 3px #b0b0b0;
}
QPushButton:hover {
    background: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #eaeaea, stop:1 #bdbdbd
    );
    border: 2px solid #5a5a5a;
}
QPushButton:pressed {
    background: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #c3c3c3, stop:1 #fdfdfd
    );
    border: 3px inset #5a5a5a;
}</string>
    </property>
    <property name="text">
     <string>显示/隐藏坐标</string>
    </property>
   </widget>
   <widget class="QWidget" name="widgetChart2" native="true">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>371</width>
      <height>271</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
    border: 2px inset #888888;
    color: #000000;
    background: #FFFFFF;
    border-radius: 10px;
    padding: 2px 8px;
    font-weight: bold;
	border-radius: 8px;
}</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>746</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="test.qrc"/>
 </resources>
 <connections/>
</ui>
